percona/config.sls

```diff
@@ -46,6 +46,16 @@ mysql_python_dep:
 {% endif %}
 
 {% for global, value in global_params.iteritems() %}
+
+{% if 'tmpdir' in global %}
+{{ value }}:
+  file.directory:
+    - user: mysql
+    - group: mysql
+    - mode: 775
+    - makedirs: True
+{% endif %}
```

Comment:
If the default value is `/tmp`, we should not change ownership to `mysql:mysql` with `0775`